.prettierignore

```diff
@@ -0,0 +1,16 @@
+# elm-package generated files
+elm-stuff
+elm.json
```

Comment:
Did these configs break when prettified?

Reply:
I'm not sure but I thought as we interact with them via a tool that will keep our formatting regular. Happy to add them back in though.

Reply:
Makes sense, but I guess add them back and see.